src/lucky_migrator/tasks/db/create.cr

```diff
@@ -1,11 +1,13 @@
-require "colorize"
-
 class Db::Create < LuckyCli::Task
   banner "Create the database"
 
+  def initialize(@quiet : Bool = false)
+    @quiet = ARGV.includes? "--quiet"
```

Comment:
That issue was fairly old and I mistakenly thought the best way would be with ARGV, but I think using just the instance var should work since in LuckyCLI we are running migrations with the class and not through the CLI

Reply:
Sorry I don't understand what you mean. How does the instance var get set?